### DISCUSSION TITLE
Recordings

### DISCUSSION THREAD

**Discussion by pyrrhus80** [hvianna/audioMotion.js#61] [General]
Hi. I really love audioMotion. I haven't found an app of it's kind that even comes close to being as amazing.
I'm curious about how the videos on the youtube channel were captured. Were they done with separate screen recording software, or is there a method of capturing the output directly from audioMotion?

**Comment by hvianna** [comment]
Thank you for your kind words! The videos were recorded with [OBS Studio](https://obsproject.com/) (free software).

**Comment by pyrrhus80** [comment]
Are there any plans to allow videos to be saved?
I'd love to watch some of the visualizations in 4K on my big screen TV - but my laptop isn't up to the task of doing it in real time at an acceptable frame rate.
